### DISCUSSION TITLE
View all issues your issues in all projects based on iteration

### DISCUSSION THREAD

**Discussion by Hannnes1** [community/community#37143] [Projects and Issues]
We have recently begun using GitHub projects, and for the most part it is working great. One thing that doesn't work great for us, however, is finding what you have to do in the current iteration. I know that this might be uncommon, but the reason for that we have this problem is that most of us are working on multiple projects at a time. It would therefore be great if there was a way to see all your assigned issues in one place, based on the iteration.

Is there a way to do that? I tried adding `iteration:@current` in the [issues](https://github.com/issues) view, but that (perhaps not unexpectedly) didn't work.

**Comment by ossanna16** [comment]
Hi there @Hannnes1 and welcome to our community! Thank you for asking a great question 🙂

**Comment by alexanderniebuhr** [comment]
@Hannnes1 I think the iteration field is just searchable in the project itself, it sounds like you want to search across multiple projects? Could you merge them all in one Project and just use a **My Sprint Work** View?

Otherwise maybe you can build an API Call for that filter.
